modules/KalturaSupport/kalturaIframeClass.php

```diff
@@ -578,6 +578,19 @@ private function getEntryTitle(){
 		return "Kaltura Embed Player iFrame";
 	}
 
+	function outputCompatibilityTags(){
+		if (isset($this->request) && !empty($this->request)) {
+			$flashvars = $this->request->getFlashVars();
+			if (isset($flashvars) &&
```

Comment:
@vladm3 the same goes for here, getFlashVars method always return an array(or a default value if you supply one), and if you look closely you will see that a few lines above where you call this method("outputCompatibilityTags") we also call "$this->request->getFlashVars()" so it is available to you also from there.